### ISSUE TITLE
how high about the mlm_acc when you are pretraining?

### ISSUE THREAD

**Issue by rxmao** [microsoft/unilm#383]
**Describe**
I use a smaller model:
    model = VisionTransformerForMaskedImageModeling(
        patch_size=16, embed_dim=384, depth=8, num_heads=6, mlp_ratio=4, qkv_bias=True,
        norm_layer=partial(nn.LayerNorm, eps=1e-6), vocab_size=8192, **kwargs)
to train,but the mlm_acc is only :
Epoch: [83]  [1990/2599]  eta: 0:06:41  lr: 0.001253  min_lr: 0.001253  mlm_acc: 0.0516 (0.0511)
How about you when you are pretraining the model?

**Comment by laojiangwei** [comment]
Epoch: [140]  [ 220/2502]  eta: 0:19:18  lr: 0.000875  min_lr: 0.000875  mlm_acc: 0.0339 (0.0368)

**Comment by addf400** [comment]
Hi @rxmao @laojiangwei , 
Our base-sized BEiT pretrain 300 epochs (block masking is applied) on imagenet-1k train set can get:
```
{"train_lr": 1.001460655613469e-05, "train_min_lr": 1.001460655613469e-05, "train_mlm_acc": 0.04961304235011339, "train_loss": 6.299410108470917, "train_loss_scale": 4096.0, "train_weight_decay": 0.050000000000000495, "train_grad_norm": 0.4236109641075134, "epoch": 299, "n_parameters": 91965776}
```
@laojiangwei , do you decrease the learning rate while using a smaller batch size?

**Comment by laojiangwei** [comment]
> Hi @rxmao @laojiangwei ,
> Our base-sized BEiT pretrain 300 epochs (block masking is applied) on imagenet-1k train set can get:
> 
> ```
> {"train_lr": 1.001460655613469e-05, "train_min_lr": 1.001460655613469e-05, "train_mlm_acc": 0.04961304235011339, "train_loss": 6.299410108470917, "train_loss_scale": 4096.0, "train_weight_decay": 0.050000000000000495, "train_grad_norm": 0.4236109641075134, "epoch": 299, "n_parameters": 91965776}
> ```
> 
> @laojiangwei , do you decrease the learning rate while using a smaller batch size?

oho, i do not decrease the learning rate, i will try again. thanks~

**Comment by donglixp** [comment]
@laojiangwei  The LR can be reduced by 2x if you would like to use bsz=512.

**Comment by addf400** [comment]
> Epoch: [140] [ 220/2502] eta: 0:19:18 lr: 0.000875 min_lr: 0.000875 mlm_acc: 0.0339 (0.0368)
> I run beit-base with default configuration, but use a small batchsize, my mlm_acc is small than you, It's so confusing about it.

My training log for epoch 140/300 for your reference: 
```
{"train_lr": 0.000871549483568734, "train_min_lr": 0.000871549483568734, "train_mlm_acc": 0.041340213759802284, "train_loss": 6.4650835299491884, "train_loss_scale": 3863.3472, "train_weight_decay": 0.050000000000000495, "train_grad_norm": 0.5984940978050232, "epoch": 140, "n_parameters": 91965776}
```
